Makefile

```diff
@@ -19,11 +19,10 @@ help:
 develop: .init-db
 	@if [[ -f ".env" ]]; then echo '.env file found. Do not create it'; else echo "PYTHONPATH=./src:$${PYTHONPATH}" > env; fi
 	pipenv sync -d
-#	pre-commit install
+	pre-commit install
 	@python manage.py init-setup --all
-	@python manage.py init-demo
 
 
 init-demo:
-	@${MAKE} init-setup
+	@${MAKE} develop
```

Comment:
I'd remove `init-demo` from `Makefile`:
 - try to not have same command Makefile/manage.py
 - do not add much value - only 1 line because develop should be removed as it should be called only the first time the development start, where `./manage.py init-demo`  can be called multiple times